### ISSUE TITLE
Profile is stored in `Local` AppData folder instead of `Roaming`

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#1473]
The user's profile on Windows is currently stored to %USERPROFILE%\AppData\Local\BraveSoftware instead of %APPDATA%. The intention/advantage of Roaming would be so that people in an enterprise setting, computer lab, university setting, or another setting with shared computers could have the same profile no matter where they log in

This would likely be important to implement for enterprise support

**Comment by bsclifton** [comment]
cc: @rebron @davidtemkin

**Comment by srirambv** [comment]
This is also an issue with offline installer. The application and user profile are both created under `Local` AppData folder where as stub installer separate the application folder into `C:\Program Files (x86)\BraveSoftware\` and user profile data into `Local` AppData folder.

**Comment by Brave-Matt** [comment]
+1's from Community:
https://community.brave.com/t/brave-installs-to-incorrect-folder-based-on-architecture-in-windows-x64/34678/4

**Comment by bsclifton** [comment]
+1 from Reddit:
https://www.reddit.com/r/brave_browser/comments/ajd396/brave_on_windows_change_from_appdatalocal_to/

**Comment by tomlowenthal** [comment]
The big risk here is that profiles don't sync; the last-used profile clobbers the rest. Honestly, I'd rather support this use case with our sync, even though that's a little way out.

**Comment by Brave-Matt** [comment]
+1 from reddit:
https://www.reddit.com/r/brave_browser/comments/chmaie/appdataroaming_instead_local/

**Comment by Neustradamus** [comment]
Dear @brave team,

Have you looked for this historical problem?

Currently, user profile is saved in the bad folder.

%LOCALAPPDATA% must be replaced by %APPDATA%.

But attention, for example, Mozilla Firefox has:
- Install folder: %PROGRAMFILES%\Mozilla Firefox
- Profile folder: %APPDATA%\Mozilla\Firefox
- Cache folder: %LOCALAPPDATA%\Mozilla\Firefox

Linked to:
- https://github.com/brave/brave-browser/issues/1473
- https://github.com/brave/brave-browser/issues/4609

Thanks in advance.

**Comment by Neustradamus** [comment]
Dear all @brave team, @bbondy, @mherrmann, @srirambv, @mkarolin, @rebron, ...

We are at the end of 2025, this VERY GOOD ticket has been done by @bsclifton himself 2018-10-10 (MORE 7 YEARS).

Can you solve it?

All simple/normal users and organization administrators would like this IMPORTANT FIX.

Your company must to respect MICROSOFT RULES WHICH EXIST SINCE WINDOWS 2000 (2000-02-17).

Thanks in advance.

Linked to:
- https://github.com/brave/brave-browser/issues/44849
- https://github.com/brave/brave-browser/issues/41854
- https://github.com/brave/brave-browser/issues/27595
- https://github.com/brave/brave-browser/issues/18711
- https://github.com/brave/brave-browser/issues/15258
- https://github.com/brave/brave-browser/issues/4609
- https://github.com/brave/brave-browser/issues/1473

**Comment by bsclifton** [comment]
We have the fix for `UserDataDir` merged - will be in the next Nightly (`1.87.42` or newer).

With that fixed, folks can set a policy for the `UserDataDir` and force it to default to roaming. For example, as a quick and dirty test, you can open regedit, navigate / create the key at `HKEY_CURRENT_USER\SOFTWARE\Policies\BraveSoftware\Brave`, then create a new string key called `UserDataDir` and set the value to `${roaming_app_data}\BraveSoftware\Brave-Browser`.  If you had installed the policy templates, you could set in the group policy editor and that should work too.

Updating the "out of the box" default to use `%APPDATA%` (instead of `%LOCALAPPDATA%`) is a bigger ask.  This is currently marked as a P5, which means it's not something we're going to look at. 

cc: @rebron in case we wanted to reprioritize this.  I'm pretty sure all the browsers I tested all install to `%LOCALAPPDATA%` also. We're not deviating from Chromium here.

**Comment by Neustradamus** [comment]
It works for each Brave Browser channels and for All Users of the machine (Not only current) at the same time?
- Brave Browser
- Brave Browser Beta
- Brave Browser Nightly

The "Cache" is separated from "User Data"?

**Comment by bsclifton** [comment]
@Neustradamus correct, the group policy is something that applies to all channels of Brave on the machine for all users. But there are variables, like `${roaming_app_data}` so that you can have a different result for each user.  Other variables here:
https://support.google.com/chrome/a/answer/9866158?visit_id=638997881433248303-507260686&p=Supported_directory_variables&rd=1

I haven't looked to see if the cache can be separated - but by default it's part of the user data.  If that's something you'd like to do, we should make a new issue for this. This issue is tracking using `Local` instead of `Roaming`.

**Comment by Neustradamus** [comment]
@bsclifton: Thanks for your answer!

Can you create a new ticket to have the "Cache" in %LOCALAPPDATA%?

Have you looked other points?
